t/unit/utils/test_collections.py

```diff
@@ -293,7 +293,7 @@ def test_update(self):
 
     def test_iterable_and_ordering(self):
         s = LimitedSet(maxlen=35, expires=None)
-        # we use a custom clock here, as time.time() does not have enough
+        # we use a custom clock here, as monotonic() does not have enough
```

Comment:
@caronc Have you read the comment about this test?
This is no longer the case with the monotonic clock.
We probably need to adjust this test to use the real clock.

Reply:
So after re-reading the test I think it is necessary to ensure ordering.
You can just remove the comment.

Reply:
done!